blogpost.html

```diff
@@ -1,162 +0,0 @@
-<!DOCTYPE html>
-<html>
-
-<head>
-    <title>Blog Post</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-    <link rel="stylesheet" href="blogpost.css" />
-</head>
-
-<body>
-
-    <main>
-
-        <section id="header">
-            <div class="title">
-                <a id="postonly" href="homepage.html">POST ONLY</a>
-                <div id="underlinetext">A PAGE WITH JUST A BLOG POST CONTENT</div>
-            </div>
-        </section>
-
-        <hr>
-
-        <section id="main">
-
-            <div class="mainimage">
-                <div id="blogpost">this is a BLOG POST only layout</div>
-                <div id="stevenphoto">
-                    <img src='https://www.stevensegallery.com/1200/300' />
-                </div>
-            </div>
-
-
-            <div class="wholepost">
-                <div id="posttitle" class="post-title-class">AND YEAH, THAT'S BASICALLY IT</div>
-                <div id="posttext">
```

Comment:
You may want to use text specific elements when wrapping content that's going to be just text. `<p>` elements respect your white-space (e.g. line-breaks, leading and trailing spaces, etc.) and are usually better than generic elements for text.